spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/publisher/PubSubPublisherOperations.java

```diff
@@ -0,0 +1,63 @@
+/*
+ *  Copyright 2017-2018 original author or authors.
```

Comment:
This is fully a new class and its birthday is exactly `2018`. The `2017` is redundant here.

I can guess that all these my comments about code style is just a noise for you. Feel free to ignore them: we can simply polish it in the subsequent commit.  Just in case you will contribute more, so that is going to be easier for us to review the next your PR.

Reply:
Fixed dates on new files to be `2018`